openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -320,11 +320,13 @@ This specification defines the following parameters for the Credential Offer obj
+  * `issuer_state`: OPTIONAL. String value created by the Credential Issuer and opaque to the Wallet that is used to bind the subsequent Authorization Request with the Credential Issuer to a context set up during previous steps. If the Wallet decides to use the Authorization Code Flow and received a value for this parameter, it MUST include it in the subsequent Authorization Request to the Credential Issuer as the `issuer_state` parameter value.
+  * `authorization_server`: OPTIONAL when a Credential Issuer is relying on multiple Authorization Servers. MUST NOT be used otherwise. String that the Wallet can use to identify the Authorization Server to used with this grant type when `authorization_servers` parameter in the Credential Issuer metadata has multiple entries. MUST match with one of the values in the `authorization_servers` array obtained from the Credential Issuer metadata.
 * Grant Type `urn:ietf:params:oauth:grant-type:pre-authorized_code`:
   * `pre-authorized_code`: REQUIRED. The code representing the Credential Issuer's authorization for the Wallet to obtain Credentials of a certain type. This code MUST be short lived and single use. If the Wallet decides to use the Pre-Authorized Code Flow, this parameter value MUST be included in the subsequent Token Request with the Pre-Authorized Code Flow.
   * `user_pin_required`: OPTIONAL. Boolean value specifying whether the AS expects presentation of a user PIN along with the Token Request in a Pre-Authorized Code Flow. Default is `false`. This PIN is intended to bind the Pre-Authorized Code to a certain transaction to prevent replay of this code by an attacker that, for example, scanned the QR code while standing behind the legitimate user. It is RECOMMENDED to send a PIN via a separate channel. If the Wallet decides to use the Pre-Authorized Code Flow, a PIN value MUST be sent in the `user_pin` parameter with the respective Token Request.
   * `interval`: OPTIONAL. The minimum amount of time in seconds that the Wallet SHOULD wait between polling requests to the token endpoint (in case the Authorization Server responds with error code `authorization_pending` - see (#token_error_response)). If no value is provided, Wallets MUST use 5 as the default.
+  * `authorization_server`: OPTIONAL when a Credential Issuer is relying on multiple Authorization Servers. MUST NOT be used otherwise. String that the Wallet can use to identify the Authorization Server to used with this grant type when `authoziation_servers` parameter in the Credential Issuer metadata has multiple entries. MUST match with one of the values in the `authorization_servers` array obtained from the Credential Issuer metadata.
```

Comment:
```suggestion
  * `authorization_server`: OPTIONAL String that the Wallet can use to identify the Authorization Server to used with this grant type when `authorization_servers` parameter in the Credential Issuer metadata has multiple entries. MUST NOT be used otherwise. MUST match with one of the values in the `authorization_servers` array obtained from the Credential Issuer metadata.
```